package com.exception.handling1;

public class ThrowDemo3 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		throw new ArithmeticException(" / by zero ");
		
//		We can not write any statement after throw keyword.
//		C.E - Unnreachable statement
//		System.out.println("Hello")

	}

}
